site/docs/getting-started/_designing/IconsTable.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement, cloneElement, useRef, useEffect } from "react";
+import { SaltProvider, Density } from "@salt-ds/core";
+
+import styles from "./IconsTable.module.css";
+
+type IconsTableProps = { children: ReactElement; caption: string };
+
+const IconsTable = ({ children, caption }: IconsTableProps) => {
+  const tableRef = useRef<HTMLTableElement>(null);
+
+  useEffect(() => {
+    // Add table caption
+    tableRef.current.createCaption().textContent = caption;
```

Comment:
I've added this in because I don't think there's a way to add a table caption in mdx which would have been a lot nicer